### DISCUSSION TITLE
Allow user to provide a custom visualization for default Tempo Trace view

### DISCUSSION THREAD

**Discussion by mkaisercross** [grafana/grafana#70876] [Explore]
I'd like to make a UI change to the default view for Tempo traces. I am fairly new to Grafana so apologies if some of the terminology I use is not correct.  

Currently if you click on a traceId link in Grafana it will opens a default Tempo view with the query area above and span tree visualization below. What I'd like to be able to do is provide a custom span tree. I'd like the same query area to be shown above just with a custom span tree below. And this new layout would be shown anytime you click traceId link from anywhere in Grafana (Tempo, Loki, or Mimir). 

To give some context here are some of the changes I have in mind:

1. Above span tree show total # of child spans with errors
2. Highlight spans in red with a status code > 400 and also spans with db errors returned
3. In each span row show a unique SVG icon pertaining to the type of span (e.g. DB, HTTP, etc)

Would something like this be possible and if not if there are any plans to offer customization of the default views in the future?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82590. Feel free to continue the discussion around this there. Thank you!
